test/org/daisy/dotify/hyphenator/impl/JHypenatorTest.java

```diff
@@ -24,6 +27,8 @@ public static String bytesToHex(byte[] bytes) {
 
     @Test
     public void testBeginEndHandling() throws HyphenatorConfigurationException {
+        Assume.assumeTrue(new File("/usr/share/hyphen/hyph_sv_SE.dic").exists());
```

Comment:
This test is very specific:
- It is for Unix based machines. (In Windows there is no "/usr/share")
- It is for Swedish.

As a result, if a developer changes something and he is not working in a Unix based environment, or he doesn't have the Swedish dictionary, this unit test will always pass. When such developer makes changes there might be an error, which appears only at a very late stadium (when you try to merge the code). I doubt whether such very specific unit tests make sense and should be incorporated in the software.

Reply:
Hi @PaulRambags 

Well, your objections make sense so the solution might be to remove the test, this test is just for the people producing information in other languages using a hyphenation engine built-in on Linux machines. On other machines, it doesn't make sense and the Swedish dictionary is used as a test case because you need to have a file to test against. I could change to use the Russian file instead but that is harder for me as I don't know that much about hyphenation in Russian.

There is no default hyphenation file that comes with the libhyphen installation.

Best regards
Daniel

Reply:
Hi again, 

I haven't installed it yet but there seems to be a dictionary in English also so that might have a hyphenation file I could create a case for. Perhaps that is more feasible?

Best regards
Daniel